.github/scripts/build_triton_wheel.py

```diff
@@ -183,8 +184,6 @@ def build_triton(
                 cwd=triton_basedir,
                 shell=True,
             )
-            cur_rocm_ver = get_rocm_version()
-            check_call(["scripts/amd/setup_rocm_libs.sh", cur_rocm_ver], cwd=triton_basedir)
```

Comment:
What's the context for removing this? Does the triton library no longer need the packaged ROCm libraries and headers?

Reply:
Triton still has bundled ROCm lib and headers. 
It is done in a different script and this change is NOT required in never branches of pytorch.